speedy-recovery-client/src/service/FhirDataMappingService.js

```diff
@@ -71,9 +71,24 @@ const fhirMapMedication = (resource, version) => {
   console.log("No FHIR version has been supplied");
 };
 
+const getChildID = currentUserResource => {
```

Comment:
Why is this function in this file? Shouldn't this be in one of the mapping adaptors? Also what if different FHR versions are used?